src/backend/defs/rest-api/objects/MessageV2.yaml

```diff
@@ -0,0 +1,59 @@
+---
+type: object
+properties:
+  attachments:
+    type: array
+    items:
+      "$ref": Attachment.json
+  body: # in future, could be only a preview of body, limited to xxx chars
+    type: string
+  date:
+    type: string
+    format: date-time
+  date_delete:
+    type: string
+    format: date-time
+  date_insert:
+    type: string
+    format: date-time
+  discussion_id:
+    type: string
+  external_references:
+    type: object
+    "$ref": ExternalReferences.json
+  identities: # to which user's local identities the message is linked to
+    type: array
+    items:
+      "$ref": LocalIdentity.json
+  importance_level:
+    type: integer
+    format: int32
+  parent_id: # The id of the Message this message is a reply to. This is primarily for drafts, but the server may support this for received messages as well by looking up the [@!RFC5322] Message-Id referenced in the In-Reply-To header and searching for this message in the user’s mail.
+    type: string
+  is_answered:
+    type: boolean
+  is_draft:
+    type: boolean
+  is_unread:
```

Comment:
we choose the 'is_unread' flag to be closer to the [jmap](http://jmap.io) specification, and because it is the default state of an incoming message. Futhermore, the 'unread' status is the status that application and user are aware of (for displaying unread count for example). Quoting jmap : « Although this may seem inconsistent at first, it actually makes more sense. The “special” status is when a message is unread (this is what clients are interested in), so like isDraft, isFlagged and isAnswered, we make the special status equal to true. It is also consistent with the need for unread counts in mailbox objects, not read counts, and makes the definition of sorting a message list the same for isFlagged and isUnread when conversations are collapsed. »